prow/plugins/updateconfig/updateconfig_test.go

```diff
@@ -410,18 +377,9 @@ func TestUpdateConfig(t *testing.T) {
 					Additions: 1,
 				},
 			},
-			existConfigMaps: map[string]kube.ConfigMap{
-				"unaffected-config": {
```

Comment:
This tested that an unaffected config was not changed. Instead of making that literal, I just check that the set of expected configs given is exactly the set of configs that were changed.